### ISSUE TITLE
Wifiphisher breaks down using the Handshake verification

### ISSUE THREAD

**Issue by pavonis9** [wifiphisher/wifiphisher#1159]
**Version**: 
git show | grep commit : 

> commit 53779384d09f3039b2ec897c7c91079c57965ae0

**Description**: 
Linux distribution : Kali Linux fully updated ( Last update August 5 2019 ) 

Wifiphisher was working normally ( more than 5 months ) and till now is still working but he stoppes when I use the handshake verification option.
I tried using 4 different wifi adapters each one have a different chipset and all are kali compatible and support monitor mode, deauthentication, AP....
By the way Wifiphisher was working always with the -hC options but not anymore.
I did even tried with the old handshakes wich they had worked with him before and still no results.

**Script output:**
root@justauser:~# wifiphisher --logging --force-hostapd -cP credentials.txt -iAM XX:XX:XX:XX:X:XX -hC pcap/justa4wayhandshake.pcap -aI wlan1 -eI wlan2
[*] Starting Wifiphisher 1.4GIT ( https://wifiphisher.org ) at 2019-08-05 22:28
[+] Timezone detected. Setting channel range to 1-13
Traceback (most recent call last):
  File "/bin/wifiphisher", line 11, in <module>
    load_entry_point('wifiphisher==1.4', 'console_scripts', 'wifiphisher')()
  File "/usr/local/lib/python2.7/dist-packages/wifiphisher-1.4-py2.7.egg/wifiphisher/pywifiphisher.py", line 693, in run
    engine.start()
  File "/usr/local/lib/python2.7/dist-packages/wifiphisher-1.4-py2.7.egg/wifiphisher/pywifiphisher.py", line 390, in start
    self.opmode.initialize(args)
  File "/usr/local/lib/python2.7/dist-packages/wifiphisher-1.4-py2.7.egg/wifiphisher/common/opmode.py", line 54, in initialize
    self._check_args(args)
  File "/usr/local/lib/python2.7/dist-packages/wifiphisher-1.4-py2.7.egg/wifiphisher/common/opmode.py", line 79, in _check_args
    is_valid_handshake_capture(args.handshake_capture):
  File "/usr/local/lib/python2.7/dist-packages/wifiphisher-1.4-py2.7.egg/wifiphisher/extensions/handshakeverify.py", line 42, in is_valid_handshake_capture
    pkts = dot11.rdpcap(handshake_path)
AttributeError: 'module' object has no attribute 'rdpcap'

**Comment by Eduardo47** [comment]
I have the same problem. The handshake verification worked ok in Kali linux, until its last update. I updated Kali yesterday and the same message reported by Pavonis9 appears.
I hope you find a solution. Thank you.

**Comment by pavonis9** [comment]
After 2 weeks of research, I finally found a solution.

The problem comes from scapy's update. And it's solved by downgrading Scapy:

`wget -O /tmp/scapy2.3.2.tar.gz https://pypi.python.org/packages/6d/72/c055abd32bcd4ee6b36ef8e9ceccc2e242dea9b6c58fdcf2e8fd005f7650/scapy-2.3.2.tar.gz; sudo pip2 install /tmp/scapy2.3.2.tar.gz`

I know this is just temporary solution, but it's the best what we've got for now.
